WcaOnRails/app/models/user.rb

```diff
@@ -568,6 +568,10 @@ def team_member?(team)
     self.current_team_members.select { |t| t.team_id == team.id }.count > 0
   end
 
+  def team_membership_details?(team)
```

Comment:
Nitpick: In Rails, methods that end with `?` should return a boolean by convention.
There is no equivalent convention (to my knowledge) for nullable methods, so calling this `team_membership_details` should be fine.

Reply:
Done.